conbench/app/benchmarks.py

```diff
@@ -112,7 +112,9 @@ def show_benchmark_cases(bname: str) -> str:
 
 class TypeUIPlotInfo(TypedDict):
     title: str
+    url_to_newest_result: str
     data_for_uplot: List[List]
+    unit: str
 
```

Comment:
The beauty of ad-hoc typing. This code is in flux, but it doesn't quickly become a gigantic mess, because of typing constraints like this.